src/index.html

```diff
@@ -53,11 +53,14 @@
   </div>
```

Comment:
The main change:
https://github.com/codenvy/dashboard/pull/6/files#diff-e344fe62242ed0193eac3bd13de1c259R46